spec/features/variable_drafts/create_variable_draft_from_variable_spec.rb

```diff
@@ -17,4 +17,35 @@
       expect(page).to have_content('Test Edit Variable Name')
     end
   end
+
+  context 'when working with the draft of a published variable' do
+    before do
+      login
+      @native_id = 'TestVariableNativeId'
+      ingest_response, _concept_response = publish_variable_draft(name: 'Test Edit Variable Name 2', native_id: @native_id)
+    end
+
+    context 'when trying to edit the draft name' do
+      before do
+        @variable_draft = create(:full_variable_draft, native_id: @native_id)
+        visit edit_variable_draft_path(@variable_draft)
+      end
+
+      it 'cannot edit the name' do
+        expect(page).to have_field('variable_draft[draft][name]', readonly: true)
+      end
+    end
+
+    context 'when trying to publish/update a variable draft with a different name' do
```

Comment:
I feel a little ambivalent about this test. On one hand, it is good to know this is supposed to be blocked and what will happen. On the other hand, this is effectively a test in MMT's code that is testing that CMR is behaving correctly. But, since it is also testing your change to `error_message` then I think we should leave it.